standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -895,13 +895,14 @@ struct AddPartitionsResult {
 
 // Request type for add_partitions_req
 struct AddPartitionsRequest {
-  1: required string dbName,
-  2: required string tblName,
-  3: required list<Partition> parts,
+  1: optional string dbName,
```

Comment:
Is there any particular reason to change the required parameters to optional?

Reply:
Ack